jscomp/super_errors/super_typecore.ml

```diff
@@ -121,9 +121,9 @@ end
 
 let reportArityMismatch ~arityA ~arityB ppf =
   fprintf ppf "This function expected @{<info>%s@} %s, but got @{<error>%s@}"
-    arityA
```

Comment:
This was swapped, not swapping it back so it's consistent with the other error messages (A is the given one, and B is the expected).